ClientApp/viking-vault/src/display_user_list/DisplayUsers.css

```diff
@@ -67,29 +72,33 @@
     margin-top: 1%;
 }
 
-#user_name{
+#user-name{
```

Comment:
you shouldn't use ids for styling, id is using for identifying an html element. Use a class instead. Same for card number below :)